src/integrationTest/java/org/opensearch/security/SecurityConfigurationTests.java

```diff
@@ -209,7 +210,28 @@ public void shouldUseSecurityAdminTool() throws Exception {
 		assertThat(exitCode, equalTo(0));
 		try(TestRestClient client = cluster.getRestClient(USER_ADMIN)) {
 			Awaitility.await().alias("Waiting for rolemapping 'readall' availability.")
-				.until(() -> client.get("_plugins/_security/api/rolesmapping/readall").getStatusCode(), equalTo(200));
+					.until(() -> client.get("_plugins/_security/api/rolesmapping/readall").getStatusCode(), equalTo(200));
+		}
+	}
+
+	@Test
+	public void shouldReloadExtensionsConfigurationFromFile() throws Exception {
+		SecurityAdminLauncher securityAdminLauncher = new SecurityAdminLauncher(cluster.getHttpPort(), cluster.getTestCertificates());
+		File config = configurationDirectory.newFile("config.yml");
+		ConfigurationFiles.createConfigFile(config);
+		int exitCode = securityAdminLauncher.updateConfig(config);
```

Comment:
Nice! This is great that you were able to show the dynamic config settings being update. 🚀